### ISSUE TITLE
Julia 1.3 support

### ISSUE THREAD

**Issue by mtfishman** [JuliaPlots/GraphMakie.jl#41]
It looks like this line: https://github.com/JuliaPlots/GraphMakie.jl/blob/649f1aec493bc57fdd3ca66e57d6b5eaaf946af4/src/beziercurves.jl#L55
is using syntax introduced in Julia 1.4:
https://github.com/JuliaLang/julia/blob/master/HISTORY.md#new-language-features-3

It seems like in general `Makie` supports Julia 1.3:
https://github.com/JuliaPlots/Makie.jl/blob/2faa3d75733689f2d3e802b1f623d0f90343005a/Project.toml#L91

We are using GraphMakie as a backend in our new tensor network visualization backend in ITensors.jl. Generally we try to support Julia 1.3 so I think it would be nice if GraphMakie did as well.

**Comment by hexaeder** [comment]
A quick search showed that I'm using `array[begin]` and `array[end]` in a few places. Feel free to open a PR to change this to `array[firstindex(array])` and so forth. Then we should also add those old Julia versions to the tests (the tests are not great at all but at least they will run plot command a few times)...

**Comment by mtfishman** [comment]
Sure, I can make a PR.

Around when I raised this issue Julia 1.6 was switched to LTS (and some big projects like [SciML](https://discourse.julialang.org/t/scimls-stance-on-the-new-julia-lts/72446) have switch to only supporting Julia 1.6), so this may not be so important, but for the time being I can still fix the issue to make it work back to Julia 1.3 since Makie support 1.3.

**Comment by hexaeder** [comment]
Yeah i was thinking the same. If you need it for `ITensors` go for it I'm happy to merge. But maybe it is a good time to get rid of the support of those ancient versions. I think with SciML raising the bounds to 1.6 there won't be a lot of 1.3 users left....
